### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by ChrisMaddock**
See https://github.com/nunit/nunit-console/pull/612. There was only this one case to fix on the framework. 🙂

### CODE REVIEW

File: src/NUnitFramework/framework/Internal/Execution/EventPump.cs [hunk 1]
```diff
@@ -191,7 +191,7 @@ private void PumpThreadProc()
             }
             catch (Exception ex)
             {
-                log.Error( "Exception in pump thread", ex );
+                log.Error("Exception in pump thread {0}", ex);
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Internal/Execution/EventPump.cs]: Any reason you are not using `ExceptionHelper` in this repo (similar to the changes in the console-repo)?

And should we also make line 188 consistent with this line and remove `\r\n` ?

**Comment by mikkelbu** [comment]
I've created #3212 to solve the problem with the Linux build on Travis.

File: src/NUnitFramework/framework/Internal/Execution/EventPump.cs (hunk 1 above)
Comment (ChrisMaddock) [reply src/NUnitFramework/framework/Internal/Execution/EventPump.cs]: Done! 🙂

**Comment by ChrisMaddock** [comment]
Thanks Mikkel, changes made. 😄

**Pull request merged**